package.json

```diff
@@ -45,7 +47,7 @@
     "eslint-plugin-promise": "6.0.0",
     "eslint-plugin-react": "7.29.4",
     "eslint-plugin-react-hooks": "4.5.0",
-    "eslint-plugin-react-native": "^4.0.0",
+    "eslint-plugin-react-native": "hsource/eslint-plugin-react-native#v4.0.0-wanderlog.2",
```

Comment:
Kannst du kurz erklären warum genau diese version verwendet werden soll?

Reply:
Steht in der Commit-Message: Die enthält einen Fix für einen Bug in dem Plugin